checks/remotesettings/collections_consistency.py

```diff
@@ -104,7 +104,11 @@ async def has_inconsistencies(server_url, auth, resource):
 
 
 async def run(server: str, auth: str) -> CheckResult:
-    resources = await fetch_signed_resources(server, auth)
+    try:
+        resources = await fetch_signed_resources(server, auth)
+    except ValueError as e:
```

Comment:
Can we maybe make this a more specific exception? `fetch_signed_resources()` already explicitly throws two different `ValueError`s before this PR, and the PR adds a third one. In addition, virtually everything built into Python might implicitly throw a `ValueError`, so if we catch `ValueError`, we don't really know what we will get.